kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -192,6 +192,15 @@ data:
     # [agent_settings.proxy_config]
     #    ignore_proxy_settings = "true"  # if this is not applied, default value is false
 
+    # Uses fluent-bit for perf and container inventory instead of fluentd
+    # Disables custom metrics
+    # [agent_settings.resource_optimization]
+    #    enabled = false
+
+    # Disables fluent-bit for perf and container inventory for Windows
+    #[agent_settings.windows_fluent_bit]
+    #    disabled = "false"
```

Comment:
instead of string, we should use boolean similar to resource_optimization

Reply:
I think it is okay. There is no consistency in the whole configmap, some places it is string and some bool. Since, we are going to use this only in case there is an issue, it should be fine